src/util.ts

```diff
@@ -22,21 +23,32 @@ import * as parse5 from 'parse5';
 import * as path from 'path';
 import {Analysis} from 'polymer-analyzer';
 
-import util = require('util');
+const {execFile: _execFile} = require('child_process');
 import _mkdirp = require('mkdirp');
 import _rimraf = require('rimraf');
+const {promisify} = require('util');
 
 import {ConvertedDocumentFilePath} from './urls/types';
 
+export interface ExecResult {
+  stdout: string;
+  stderr: string;
+}
+
+/**
+ * Helper promisified "execFile" library function.
+ */
+export const execFile = promisify(_execFile);
```

Comment:
The `exec()` helper below should be the only wrapper we expose. No worries about making another change, I'll fix this on merge.